### PR TITLE
feat(sticky-header): add app-demo for sticky-header

### PR DESCRIPTION
**Pull request by sllethe**

### CODE REVIEW

File: src/demo-app/sticky-header/sticky-header-demo.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+
+@Component({
+    moduleId: module.id,
+    selector: 'sticky-header-demo',
+    templateUrl: 'sticky-header-demo.html',
+    styleUrls: ['sticky-header-demo.css'],
+})
+
```
Comment (tinayuangao) [inline src/demo-app/sticky-header/sticky-header-demo.ts]: nit: remove the empty lines

File: src/demo-app/sticky-header/sticky-header-demo.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import {Component} from '@angular/core';
+import {NgModule} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {StickyParentDirective, StickyHeaderDirective} from '@angular/material';
+export {StickyParentDirective, StickyHeaderDirective};
```
Comment (tinayuangao) [inline src/demo-app/sticky-header/sticky-header-demo.ts]: Don't need to export them

File: src/demo-app/sticky-header/sticky-header-demo.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import {Component} from '@angular/core';
+import {NgModule} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
```
Comment (tinayuangao) [inline src/demo-app/sticky-header/sticky-header-demo.ts]: This and `NgModule` import can be removed

File: src/demo-app/sticky-header/sticky-header-demo.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+@Component({
+    moduleId: module.id,
+    selector: 'sticky-header-demo',
+    templateUrl: 'sticky-header-demo.html',
+    styleUrls: ['sticky-header-demo.css'],
+})
+
+
+export class StickyHeaderDemo {
```
Comment (tinayuangao) [inline src/demo-app/sticky-header/sticky-header-demo.ts]: Remove constructor 
` export class StickyHeaderDemo {}`

File: src/demo-app/sticky-header/sticky-header-demo.html [hunk 5]
```diff
@@ -0,0 +1,107 @@
+
+<div class="demo-sticky">
+    <div class="centered" cdk-scrollable>
+        <h1>Sticky-header Demo</h1>
+        <div id="testFor" class="container">
+
+            <div id="theStickyHeaderLalala" style="margin: 0; border: 0; padding: 0;">
```
Comment (tinayuangao) [inline src/demo-app/sticky-header/sticky-header-demo.html]: Move styles to css class if possible.
Remove the id if it's not needed

File: src/demo-app/sticky-header/sticky-header-demo.scss [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+.rside3 {
+  background: #c2eae5;
+  -webkit-appearance: none;
+  padding: 20px;
+  //width: 100%;
+}
+
+
+h2 {
```
Comment (tinayuangao) [inline src/demo-app/sticky-header/sticky-header-demo.scss]: add class for these elements? `h2`, `p`, and `div`

**Comment by sllethe** [comment]
closed by accident...

File: .firebaserc [hunk 7]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (tinayuangao) [inline .firebaserc]: Revert this file?

File: src/demo-app/sticky-header/sticky-header-demo.scss [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+  //width: 100%;
+}
+
+h2 {
+  font-size: 20px;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+}
+
+p {
```
Comment (tinayuangao) [inline src/demo-app/sticky-header/sticky-header-demo.scss]: Can we use class for `p` `h2` `div` `body` `header` `*`?

Comment (sllethe) [inline src/demo-app/sticky-header/sticky-header-demo.scss]: Put class for `p h2 div body header *` into '`democlass`' class :+1:

File: .firebaserc (hunk 7 above)
Comment (sllethe) [inline .firebaserc]: :+1:  Reverted

**Pull request closed without merging**